kili/mutations/asset/__init__.py

```diff
@@ -289,3 +290,38 @@ def generate_variables(batch):
                                               generate_variables,
                                               GQL_ADD_ALL_LABELED_ASSETS_TO_REVIEW)
         return format_result('data', results[0])
+
+    @Compatible(['v2'])
+    @typechecked
+    def send_back_assets_to_queue(
```

Comment:
I agree the name "send_assets_back_to_queue" seems best to me but the resolver is "sendBackAssetsToQueue", that's why I used "send_back_assets_to_queue". Wdyt ?

Reply:
@p-desaintchamas last time that I was in a call with Damien to speak about this, we agreed that we are sending a label back to queue and not an asset. I see that the resolver and the app rather talk about assets sent back to queue. How did you manage this for the final users ?

Reply:
I don't see how we could send a label back to queue? I mean, here, we changed the asset_status from "REVIEWED" to "ONGOING" (label_type remains "REVIEW"). For a label, it would mean changing the label_type from "REVIEW" to "DEFAULT" I guess, but what would you do about the asset_status?